File: root_easysetup/programs/VB/Samples/WS_sample/WSIFType_sample/TestParameterValue.vb
'**********************************************************************************
'* フレームワーク・テストクラス
'**********************************************************************************

'**********************************************************************************
'* クラス名        ：TestParameterValue
'* クラス日本語名  ：テスト用の引数クラス
'*
'* 作成日時        ：－
'* 作成者          ：sas 生技
'* 更新履歴        ：
'*
'*  日時        更新者            内容
'*  ----------  ----------------  -------------------------------------------------
'*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
'*
'**********************************************************************************

' System
Imports System

' ベースクラス
Imports Touryo.Infrastructure.Business.Util
Imports Touryo.Infrastructure.Business.Common

Namespace WSIFType_sample
	''' <summary>
	''' TestParameterValue の概要の説明です
	''' </summary>
	''' <remarks>シリアライズ可能にする（WS対応）</remarks>
	<Serializable()> _
	Public Class TestParameterValue
		Inherits MyParameterValue
		''' <summary>汎用エリア</summary>
		Public Obj As Object

		''' <summary>ShipperID</summary>
		Public ShipperID As Integer

		''' <summary>CompanyName</summary>
		Public CompanyName As String

		''' <summary>Phone</summary>
		Public Phone As String

		''' <summary>OrderColumn</summary>
		Public OrderColumn As String

		''' <summary>OrderSequence</summary>
		Public OrderSequence As String

		#Region "コンストラクタ"

		''' <summary>コンストラクタ</summary>
        Public Sub New(ByVal screenId As String, ByVal controlId As String, ByVal methodName As String, ByVal actionType As String, ByVal user As MyUserInfo)
            ' Baseのコンストラクタに引数を渡すために必要。
            MyBase.New(screenId, controlId, methodName, actionType, user)
        End Sub

		#End Region
	End Class
End Namespace

File: root/programs/C#/Samples/WebApp_sample/ProjectX_sample/AppCode/sample/Common/TestParameterValue.cs
//**********************************************************************************
//* フレームワーク・テストクラス
//**********************************************************************************

//**********************************************************************************
//* クラス名        ：TestParameterValue
//* クラス日本語名  ：テスト用の引数クラス
//*
//* 作成日時        ：－
//* 作成者          ：sas 生技
//* 更新履歴        ：
//*
//*  日時        更新者            内容
//*  ----------  ----------------  -------------------------------------------------
//*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
//*
//**********************************************************************************

// ベースクラス
using Touryo.Infrastructure.Business.Util;
using Touryo.Infrastructure.Business.Common;

namespace MyType
{
    /// <summary>
    /// TestParameterValue の概要の説明です
    /// </summary>
    public class TestParameterValue : MyParameterValue
    {
        /// <summary>汎用エリア</summary>
        public object Obj;

        /// <summary>ShipperID</summary>
        public int ShipperID;

        /// <summary>CompanyName</summary>
        public string CompanyName;

        /// <summary>Phone</summary>
        public string Phone;

        /// <summary>OrderColumn</summary>
        public string OrderColumn;

        /// <summary>OrderSequence</summary>
        public string OrderSequence;

        #region コンストラクタ

        /// <summary>コンストラクタ</summary>
        public TestParameterValue(string screenId, string controlId, string methodName, string actionType, MyUserInfo user)
            : base(screenId, controlId, methodName, actionType, user)
        {
            // Baseのコンストラクタに引数を渡すために必要。
        }

        #endregion
    }
}
